src/vpn-api/utils/index.ts

```diff
@@ -20,5 +20,5 @@ export { clients };
 export { HAProxy } from './haproxy';
 export { Netmask } from './netmask';
 export { VpnManager } from './openvpn';
-export { request } from './request';
+export { pooledRequest as request } from './request';
```

Comment:
Can you avoid renaming this when re-exporting please, it means that further down we can't tell that it's pooled
```suggestion
export { pooledRequest } from './request';
```
alternatively downstream can just import directly from the `./request` module